resources/group.py

```diff
@@ -177,26 +204,26 @@ def put(self, name):
         if group is None:
             raise NotFound("No such group")
 
-        params = dict(capabilities=obj.data.get("capabilities", group.capabilities),
-                      name=obj.data.get("name", name))
-
-        if params["name"] != name and group.immutable:
-            raise Forbidden("Renaming group not allowed - group is immutable")
+        if obj.data["name"] is not None:
```

Comment:
```suggestion
        if obj.data["name"]:
```

Otherwise this could result in group name being set to `""` which sounds like a really bad idea.

EDIT: this might not be the case, look note at `GroupUpdateRequestSchema` validator.

Reply:
Yup, appropriate validation should be done at Schema level - fixed :+1: